doc/troubleshooting.md

```diff
@@ -10,14 +10,14 @@ If any path-related problem still occurs, please try the following approaches:
 * try to run with `sudo -E kube-spawn ...` to pass normal-user's env variables
 * try to run with pre-defined `GOPATH` or `CNI_PATH`, for example `sudo GOPATH=/home/myuser/go CNI_PATH=/home/myuser/go/bin kube-spawn ...`
 
-## SELinux and firewalld
```

Comment:
We're doing this programmatically now, should we still keep the iptables part of this section?